community/sshfs/post-install

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh -e
+
+ln -sf /usr/bin/sshfs /usr/bin/mount.sshfs
+ln -sf /usr/bin/sshfs /usr/bin/mount.fuse.sshfs
```

Comment:
This needs to happen in `build`.

Example: `ln -s /usr/bin/sshfs "$1/usr/bin/mount.sshfs"`